webpack.config.coffee

```diff
@@ -20,7 +20,7 @@ else
 module.exports =
   cache: true
 
-  devtool: 'source-map' # Always include source maps!
+  devtool: '#inline-source-map' # Always include source maps!
 
```

Comment:
Hm, okay, I'll test on dev.  If that's true though, would we rather disable source maps in other environments other than dev, or keep them, and just not use chrome to debug?  I believe the source maps are fine in firefox.